readthedocs/search/parsers.py

```diff
@@ -413,7 +414,9 @@ def _process_content(self, page, content):
         body = self._get_main_node(html)
         title = ""
         sections = []
+        main_content_hash = None
         if body:
+            main_content_hash = hashlib.md5(body.html.encode()).hexdigest()
```

Comment:
By running in the search indexing, we're not querying S3 to get these files, right? We're processing the manifest when they are on disk on the builder? That's a huge win.

Reply:
We query S3 for search indexing, but for generating the manifest we are not doing any extra calls.

Reply:
He, I thought the same a few minutes ago and wrote https://github.com/readthedocs/readthedocs.org/pull/11643#issuecomment-2405245920